### ISSUE TITLE
Which version of OpenGL this project use?

### ISSUE THREAD

**Issue by shujaatak** [zotya701/Cpp-OpenGL-GUI#1]
I searched for the shaders inside the repository but I didn't find any, does it use old immediate mode OpenGL? If yes, then any plans for the update please?

**Comment by zoltan-hadur** [comment]
Yes, it uses immediate mode. There is no plan for that in the near future.
For the version, glGetString(GL_VERSION) returns "4.6.13570 Compatibility Profile Context 19.7.1 26.20.13001.9005". FreeGLUT is the 3.0.0 MSVC one from https://www.transmissionzero.co.uk/software/freeglut-devel/.

**Comment by shujaatak** [comment]
Thank you so much for such a detailed response. 
In fact, I am trying to learn how to create gui widgets in OpenGL 4.5 so I am searching for a simple tutorial or project that may get me started.

**Comment by shujaatak** [comment]
By the way I like the way you have made smooth animations of the windows!
Great job!
